pull_request_template.md

```diff
@@ -15,6 +15,7 @@ https://invis.io/RYA3XN5WP#/273832670_Homepage_2-2_E
 
 The following are ALL required for the PR to be merged:
 - [ ] Frontend review completed
+- [ ] All componentWillReceiveProps, componentWillMount, and componentWillUpdate in relevant child/parent components/containers replaced with [future compatible life-cycle methods](https://reactjs.org/blog/2018/03/27/)update-on-async-rendering.html) per [JIRA item 3339](https://federal-spending-transparency.atlassian.net/browse/DEV-3339)
 - [ ] Design review (if applicable)
```

Comment:
`https://reactjs.org/blog/2018/03/27/)update-on-async-rendering.html` parens before the `u` in update